.goreleaser.brew.yml

```diff
@@ -20,4 +20,15 @@ brew:
     - aws-iam-authenticator
 
   custom_block: |
-    head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd64.tar.gz"
+    head "https://github.com/weaveworks/eksctl/releases/download/latest_release/eksctl_Darwin_amd65.tar.gz"
```

Comment:
`65`

Reply:
👍 rebased

Reply:
It's still changed to `amd65`, that can't be right. accidental vim ctrl-a?

Reply:
Ahhh sorry. Totally misunderstood your original comment. 

Good catch - must have been ctrl-a (which is also my tmux bind).

Fixed.